src/ec/packages/ec-component-timeline/ecl-timeline.html.twig

```diff
@@ -8,10 +8,10 @@ Parameters:
     - "from": (integer) (default: items.length) Item index after which to start hiding timeline items
     - "to": (integer) (default: items.length) Item index after which to resume displaying timeline items
   - "items" (array) (default: []):
-    - "id": (string) (default: '')
+    - "id": (string) (optional) (default: '')
     - "label": (string) (default: '')
     - "title": (string) (default: '')
-    - "content": (block) (default: '')
+    - "content": (string) (optional) (default: '')
```

Comment:
Won't it be an issue to define the content as a string only?
According to the usage page, users can put other type of content in it (media for instance)

Reply:
well, i'd say that in these docs on all our templates we refer to the data type, whatever this variable will contain it's going to be used as a string, not an array, not an object, not a boolean, in other words. But you are right that maybe we should treat this, as the other variables that are "allowed" or supposed to contain html (the ones that we were rendering with `|raw`) in a different way, because `string` in those cases might be confused with "plain text", we could say something like `html markup marked as a safe string` to somehow specify that it would be otherwise escaped in an environment with the twig autoescape on.